tests/core_tests.py

```diff
@@ -147,9 +147,14 @@ def test_slices(self):
         for slc in db.session.query(Slc).all():
             urls += [
                 (slc.slice_name, 'slice_url', slc.slice_url),
-                (slc.slice_name, 'slice_id_endpoint', '/caravel/slices/{}'.format(slc.id)),
                 (slc.slice_name, 'json_endpoint', slc.viz.json_endpoint),
                 (slc.slice_name, 'csv_endpoint', slc.viz.csv_endpoint),
+                (slc.slice_name, 'slice_id',
```

Comment:
no need to test these 3 endpoints for all slices, let's pick one and write another unit test that tests using `assertRedirects`